glam/api/management/commands/import_glean_aggs.py

```diff
@@ -114,12 +120,12 @@ def import_file(self, tmp_table, fp, app_id):
                 """
                 cursor.copy_expert(sql, tmp_file)
 
-        log(app_id, "  Inserting data from temp table into aggregation tables.")
+        log(app_id, " Inserting data from temp table into aggregation tables.")
```

Comment:
I think you left an extra space here.
```suggestion
        log(app_id, "Inserting data from temp table into aggregation tables.")
```